main.sh

```diff
@@ -20,7 +20,7 @@ function startXvfb {
     if ! isXvfbRunning; then
         rm -f /tmp/.X1-lock
         echo "Starting Xvfb"
-        Xvfb :1 -screen 0 1024x768x24 &
+        Xvfb :1 -screen 0 1024x768x24 -nolisten linux &
```

Comment:
I think you mean
`-nolisten unix`

Reply:
Yes ! my bad, it is `-nolisten unix
`
This argument tells Xvfb to not start the unix socket which is causing the `_XSERVTransmkdir: Owner of /tmp/.X11-unix should be set to root` error and all errors related to the socket wich is not necessary.